uahf-technical-spec.md

```diff
@@ -199,6 +199,13 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-7 Difficulty adjustement in case of hashrate drop
```

Comment:
"adjustement" in requirement title is still spelled wrong

Thanks for adding a rationale.